nixos/attestation-server/module.nix

```diff
@@ -64,6 +94,22 @@ in
 
       serviceConfig = {
         ExecStart = "${cfg.package}/bin/AttestationServer";
+        ExecStartPre = let
+          values = concatStringsSep ", " [
+            "('emailUsername', '${cfg.emailUsername}')"
+            "('emailPassword', TRIM(readfile('%S/attestation/emailPassword'), char(10)))"
+            "('emailHost', '${cfg.emailHost}')"
+            "('emailPort', '${toString cfg.emailPort}')"
+          ];
+        in optionals (cfg.emailPasswordFile != null) [
+          "+${pkgs.coreutils}/bin/install -m 0600 -o %N -g %N ${cfg.emailPasswordFile} %S/attestation/emailPassword"
+          ''${pkgs.sqlite}/bin/sqlite3 %S/attestation/attestation.db "INSERT OR REPLACE INTO Configuration VALUES ${values}"''
+          "${pkgs.coreutils}/bin/rm -f %S/attestation/emailPassword"
+        ];
+
+        # When sending TERM, e.g. for restart, AttestationServer fails with
+        # this exit code.
+        SuccessExitStatus = [ 143 ];
```

Comment:
This is not great but I don't know what else to do about this problem.  I couldn't find a comprehensive list of exit codes for AttestationServer anywhere.